hardware/raspberrypi/datasheets.md

```diff
@@ -0,0 +1,14 @@
+## Datasheets
+
+### SoC's
+
+The following datasheets are available for the SoC (System On Chip - the main processing chip) devices used on the Raspberry Pi platform.
+
+[BCM2711](bcm2711/rpi_DATA_2711_1p0.pdf) - The SoC used on the Raspberry Pi 4.
+
+[BCM2835](bcm2735/BCM2835-ARM-Peripherals.pdf) - The SoC used on the Raspberry Pi 0 and 1. This datasheet is also applicable to the Raspberry Pi 2 and 3, with the BCM2836 and BC2837 SoC's respectively, as the Videocore GPU is the same.
```

Comment:
BCM2835 was released before BCM2711, so I'd list it first. :man_shrugging: 
Also, I'm assuming the `bcm2735` in the URL is wrong? :laughing:

Reply:
Weird, I thought I checked all the links...